src/@ionic-native/plugins/user-agent/index.ts

```diff
@@ -0,0 +1,73 @@
+import {Injectable} from '@angular/core';
+import {Plugin, Cordova, IonicNativePlugin} from '@ionic-native/core';
+
+/**
+ * @name User-Agent
```

Comment:
This should be `User Agent` without the dash in between. Was this automatically generated by the `gulp plugin:create` command?